site/app/templates/EditSecondaryEmailForm.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-secondary-email-form{% endblock %}
+{% block title %}Edit Secondary Email{% endblock %}
+{% block body %}
+    <p>Please fill in below to change your secondary email and configure whether or not you would like
+    us to send emails to it in addition to your normal email address.</p>
+    <div class="flex-col" id="user_settings_modal">
```

Comment:
CSS Ids and classes should use dashes 
https://submitty.org/developer/coding_style_guide/css

Reply:
Yes I know Ids should use dashes. The reason I used specifically that is because the other two pop up forms in user profile used the exact same id. I kept it so I could keep any formatting already there. Should I change this for the others?

Reply:
If you need the id to be a specific way to work with existing code then its fine for now but new code should try and use our style guides even older code does not follow it on the same pages.